components/bl-worldClock-component/src/views/digital-types/digital-clock-flip.js

```diff
@@ -0,0 +1,126 @@
+import { useEffect } from 'react';
+
+import { useObjectState } from '../helpers';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const Formats = {
+  AM  : 'AM',
+  PM  : 'PM',
+  AMPM: 'ampm',
+  HOUR: 'hour',
+};
+const LimitValues = {
+  MAX_HOUR       : 23,
+  MAX_TIME_UNIT  : 59,
+  MIN_DIGIT_VALUE: 10,
+};
+const FOLD = 'fold';
+const UNFOLD = 'unfold';
+
+export function DigitalClockFlip({ time, displaySeconds }) {
+  const { hour, minute, second, isAmpm, ampm } = time;
+
+  const [state, setState] = useObjectState({
+    hour, minute, second, ampm,
+    hourShuffle  : true,
+    minuteShuffle: true,
+    secondShuffle: true,
+    ampmShuffle  : true,
+  });
```

Comment:
Could we write like this?

```suggestion
  const [ {hour, minute, second, ampm, hourShuffle, minuteShuffle, secondShuffle, ampmShuffle },setState]=seObjectState({
    ...time,
    hourShuffle  : true,
    minuteShuffle: true,
    secondShuffle: true,
    ampmShuffle  : true,
  });
```

And below and in components change

`hour` -> `time.hour` and `state.hour` - `hour`